include/poketch_data.h

```diff
@@ -50,19 +50,21 @@ enum PoketchScreenColor {
 };
 
 enum {
```

Comment:
Why are these values in an enum? They don't seem related to each other.

Reply:
I wasn't really sure how to handle these constants, since the style guide says to prefer enums over #defines whenever possible.